src/DependencyInjection/MakerExtension.php

```diff
@@ -48,6 +52,14 @@ public function load(array $configs, ContainerBuilder $container): void
             ->getDefinition('maker.maker.make_functional_test')
             ->setDeprecated(...$deprecParams);
 
+        /**
+         * @deprecated remove this block when removing make:subscriber
+         */
+        $deprecParamsSubscriber = method_exists(Definition::class, 'getDeprecation') ? ['symfony/maker-bundle', '1.51', self::SUBSCRIBER_MAKER_DEPRECATION_MESSAGE] : [true, self::SUBSCRIBER_MAKER_DEPRECATION_MESSAGE];
+        $container
+            ->getDefinition('maker.maker.make_subscriber')
+            ->setDeprecated(...$deprecParamsSubscriber);
```

Comment:
Actually, `getDeprecation()` has now existed for a long time and exists in every version that Maker supports. So can you simplify this - and while you're can you simplify the code above also for the test stuff?